experimenter/experimenter/jetstream/client.py

```diff
@@ -65,8 +65,8 @@ def validate_metadata(metadata_json):
 
 def get_metadata(slug):
     filename = f"metadata_{slug}.json"
-    path = os.path.join(METADATA_FOLDER, filename)
-    return validate_metadata(load_data_from_gcs(path))
+    path = Path(METADATA_FOLDER) / filename
+    return validate_metadata(load_data_from_gcs(str(path)))
```

Comment:
This should be updated to take a Path

Reply:
In theory but refactors like that I'd consider a followup.